src/public/govuk/stylesheets/govuk-template.css

```diff
@@ -310,10 +310,7 @@ button:focus {
     line-height: 1;
     text-decoration: none;
     text-rendering: optimizeLegibility;
-    border-bottom: 1px solid transparent;
-    background: url(images/gov.uk_logotype_crown.png?0.22.3) no-repeat;
-    background-size: 36px 32px;
-    background-position: 0 0; }
```

Comment:
How come it's not required anymore?

Reply:
Before, the logic for displaying the logo image was implemented using CSS **background** properties within the stylesheet. However, upon reviewing the code, it was determined that using CSS for this purpose was not in alignment with HTML standards. Additionally, the use of **visibility: hidden**; on an <img> tag in HTML is considered incorrect.

As a result, it was decided  @sdh100shaun  to adhere to HTML standards and best practices by displaying the logo image using an <img> tag or SVG directly in the HTML markup. This ensures semantic correctness and better accessibility for screen readers and other assistive technologies.

Additionally, with the removal of the background image, there is no longer a need for background-related CSS properties, so they were also removed for clarity and to reduce unnecessary code.

Reply:
> Additionally, the use of visibility: hidden; on an  tag in HTML is considered incorrect.

What element is `visibility: hidden` applied to and why is it considered incorrect? 

> As a result, it was decided by us / @sdh100shaun to adhere to HTML standards and best practices by displaying the logo image using an  tag or SVG directly in the HTML markup. This ensures semantic correctness and better accessibility for screen readers and other assistive technologies.

Screen readers aren't interested in a crown logo (be it in SVG or CSS background-image) so it can't be better or worse for a screen reader, nor is it more semantically correct.

I'm not disagreeing with the approach I can see it better aligns with how GDS are doing it; I don't believe this PR includes all the code it should include. Unused images? Unused CSS?

Reply:
Thanks, As part of this PR, I have removed all unused images and CSS files that were related to this task.

Reply:
[
![Screenshot 2024-02-07 at 10 29 45](https://github.com/dvsa/rsp-public-portal/assets/151522199/e30f0b9e-e7a4-44b2-9f7a-88835df6a941)
]
I've included a screenshot as a reference for you to see how the logo was previously placed within the <a> tag as a background image, with the <img> tag set to visibility: hidden.

Reply:
PR looks better! Thanks!